.github/workflows/ci.yaml

```diff
@@ -34,4 +34,4 @@ jobs:
                   dependency-versions: ${{ matrix.dependency-versions }}
 
             - name: Run PHPUnit tests
-              run: ./vendor/bin/simple-phpunit
+              run: ./vendor/bin/phpunit
```

Comment:
I think this should be reverted. We still need to use simple-phpunit, but if we want to use a newer PHPUnit version - we can specify the exact version with SYMFONY_PHPUNIT_VERSION in phpunit.xml.dist .